sympy/integrals/manualintegrate.py

```diff
@@ -342,7 +342,10 @@ def _parts_rule(integrand, symbol):
     # log, inverse trig, algebraic (polynomial), trigonometric, exponential
     def pull_out_polys(integrand):
         integrand = integrand.together()
-        polys = [arg for arg in integrand.args if arg.is_polynomial(symbol)]
+        polys = [arg for arg in integrand.args if (arg.is_polynomial(symbol) or \
+                                                   (isinstance(arg, sympy.Pow) and \
```

Comment:
Like is_polynomial, it would be nice if `(x**a).is_rational()` evaluates to `True`, but `(a**x).is_rational()` evaluates to `False` where `x` is a symbol and `a` is a constant. I think that would be most useful for the integral rules. You are right, that `a = -1` should be allowed